warehouse/query-core/src/main/java/datawave/query/language/builder/jexl/JexlTreeBuilder.java

```diff
@@ -63,7 +68,7 @@ public class JexlTreeBuilder extends QueryTreeBuilder {
             new DateFunction(), new OccurrenceFunction(), new MatchesInGroupFunction(), new MatchesInGroupLeft(), new GetAllMatches(),
             new MatchesAtLeastCountOf(), new Jexl(), new TimeFunction(), new AtomValuesMatchFunction(), new Options(), new Unique(), new GroupBy(),
             new EvaluationOnly(), new UniqueByDay(), new UniqueByHour(), new UniqueByTenthOfHour(), new UniqueByMonth(), new UniqueByYear(),
-            new UniqueByMinute(), new NoExpansion(), new Compare(), new ExcerptFields()};
+            new UniqueByMinute(), new NoExpansion(), new Compare(), new ExcerptFields(), new Sum(), new Min(), new Max(), new Average(), new Count()};
```

Comment:
These should be added to allowedQueryFunctions in QueryLogicFactory.xml as well.

Reply:
Will do.

Reply:
After double-checking, these were already added to allowedQueryFunctions in QueryLogicFactory.xml.

Reply:
I noticed that as well and forgot to delete this comment.  My apologies.